doc/source/whatsnew/v0.24.0.txt

```diff
@@ -236,6 +236,10 @@ If installed, we now require:
 | scipy           | 0.18.1          |          |
 +-----------------+-----------------+----------+
```

Comment:
you can add pyarrow to the table above with a min of 0.4.1 (you comment is ok to leave)